views/index.erb

```diff
@@ -0,0 +1,24 @@
+
+<% if @user %>
+  <h1>Welcome, <%= @user.name %></h1>
+  <p>Enter your message:</p>
+  <form action="/message" method="post">
+    <input type="text" name="content" size=100 placeholder="Your message">
+    <input type="submit" value="Submit">
+  </form>
+<% else %>
+  <h1> Welcome to Chitter</h1>
+  <p>You need to sign up to post peeps.</p>
+  <form action="/users/new" method="get">
+    <input type="submit" value="Sign up">
+  </form>
+<% end %>
+
+<p>Previous messages:</p>
+<ul>
+  <% @chitter.each do |peep| %>
+    <li class="peep" id="peep-<%= peep.id %>">
+        <%= peep.content %> (<%= peep.timestamp %>)
+    </li>
+  <% end %>
+</ul>
```

Comment:
You might want to think about using separate views for different parts of your functionality as well as layout files. This way you reduce the amount of logic you need in your views while still providing good user interaction.

More info here: 
http://ruby-log.blogspot.com/2013/02/sinatra-layouts.html